docs/UserGuide.md

```diff
@@ -402,6 +402,11 @@ Furthermore, certain edits can cause the application to behave in unexpected way
 --------------------------------------------------------------------------------------------------------------------
 <div style="page-break-after: always;"></div>
 
+## Glossary
+* **CLI**: Command-line interface where you interact with the system using your keyboard.
```

Comment:
Perhaps adding a comma behind `Command-line interface, where ...`, so that Ms Dara can't reduce our grammar mark 😭